src/main.go

```diff
@@ -77,6 +78,14 @@ func Cmd() *cobra.Command {
 		"address to forward requests to",
 	)
 
+	cmd.PersistentFlags().StringVarP(
```

Comment:
do we still need the websocket flag if we are not listening to new heades anymore?

Reply:
Removed it.